app/Models/Theme.php

```diff
@@ -0,0 +1,79 @@
+<?php
+
+namespace App\Models;
+
+use A17\Twill\Models\Behaviors\HasMedias;
+use A17\Twill\Models\Behaviors\HasPosition;
+use A17\Twill\Models\Behaviors\HasRevisions;
+use A17\Twill\Models\Behaviors\HasTranslation;
+use A17\Twill\Models\Behaviors\Sortable;
+use A17\Twill\Models\Model;
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+
+class Theme extends Model implements Sortable
+{
+    use HasFactory, HasMedias, HasPosition, HasRevisions, HasTranslation;
+
+    protected $fillable = [
+        'published',
+        'position',
+        'title',
+        'intro',
+    ];
+
+    public $translatedAttributes = [
+        'title',
+        'intro',
+    ];
+
+    public $mediasParams = [
+        'icon' => [
+            'default' => [
+                [
+                    'name' => 'default',
+                    'ratio' => 1,
+                    'minValues' => [
+                        'width' => 76,
+                        'height' => 76,
+                    ],
+                ],
+            ],
+        ],
+        'cover' => [
+            'default' => [
+                [
+                    'name' => 'default',
+                    'ratio' => 0,
+                    'minValues' => [
+                        'width' => 1125,
+                        'height' => 1500,
+                    ],
+                ],
+            ],
+        ],
+        'cover_home' => [
+            'default' => [
+                [
+                    'name' => 'default',
+                    'ratio' => 0,
+                    'minValues' => [
+                        'width' => 1125,
+                        'height' => 1500,
+                    ],
+                ],
+            ],
+        ],
+        'bgs' => [
```

Comment:
This seems like an unnecessary abbreviation. Please make this `background` instead.

Reply:
Good call. I had initially used `bgs` to mirror the output JSON.